spec/features/user_creates_voter_spec.rb

```diff
@@ -35,7 +35,7 @@
 
       visit new_voter_path
 
-      click_button 'Submit'
+      click_button 'Add this voter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.